tensorflow/core/kernels/convex_hull_op_test.cc

```diff
@@ -153,4 +153,5 @@ TEST_F(ConvexHullOpTest, Error_InputDimMustBe2) {
 }
 
 }  // namespace
-}  // namespace tensorflow
\ No newline at end of file
+
```

Comment:
One last minor nit please (since we need another commit for the golden files anyway). Ends of namespaces don't need to be separated by a blank line. Would you mind removing this line? (But please keep the newline at the end of file.)
```c++
}  // namespace
}  // namespace tensorflow
```

Reply:
removed the redundant line